index.bs

```diff
@@ -306,20 +306,24 @@ The <dfn attribute for="DeviceOrientationEvent">absolute</dfn> attribute must re
 The <dfn method for="DeviceOrientationEvent">requestPermission(<var>absolute</var>)</dfn> method steps are:
 
  1. Let <var>global</var> be the <a>current global object</a>.
- 1. If <a>this</a>'s <a>relevant global object</a> does not have <a>transient activation</a>, return <a>a promise rejected with</a> a "{{NotAllowedError}}" {{DOMException}}.
+ 1. Let <var>hasTransientActivation</var> be true if <a>this</a>'s <a>relevant global object</a> has <a>transient activation</a>, and false otherwise.
```

Comment:
shouldn't it be "or false, otherwise" ? or "or else false"

Reply:
The "[...], and/or $SOMETHING otherwise" idiom is used in several specs, they just don't agree on whether to use "and" or "or".